src/bootstrap/src/core/build_steps/llvm.rs

```diff
@@ -624,12 +621,22 @@ impl Step for Llvm {
     }
 }
 
+pub fn get_llvm_version(builder: &Builder<'_>, llvm_config: &Path) -> String {
+    command(llvm_config).arg("--version").run_capture_stdout(builder).stdout().trim().to_owned()
+}
+
+pub fn get_llvm_version_major(builder: &Builder<'_>, llvm_config: &Path) -> u8 {
+    let version = get_llvm_version(builder, llvm_config);
+    let major_str = version.split('.').next().expect("Failed to parse LLVM version");
```

Comment:
Nit (style): maybe `split_once` instead of `split().next()`?

Reply:
Sure